backend/server/src/service/ratings.rs

```diff
@@ -0,0 +1,164 @@
+use crate::models::campaign::Campaign;
+use crate::models::error::ChaosError;
+use crate::models::organisation::{Member, MemberList, OrganisationDetails, OrganisationRole};
+use chrono::{DateTime, Utc};
+use snowflake::SnowflakeIdGenerator;
+use sqlx::{Pool, Postgres, Transaction};
+use std::ops::DerefMut;
+use uuid::Uuid;
+
+/// Any member of the organisation that owns the campaign is an application
+/// viewer, because all members are either directors or execs (TODO: might be
+/// changed in the future).
+pub async fn assert_user_is_application_reviewer_admin_given_application_id(
```

Comment:
This whole thing might not be needed yet because if a person is an organisation member, they are also an application reviewer, so this authZ check should be covered by the function `assert_user_is_organisation_member` below

Reply:
But we can keep it, and update the internal logic later once we add Organisation roles